### ISSUE TITLE
Unable to fine the dataset of PICS_GWAS_SNPS.gr.rds

### ISSUE THREAD

**Issue by Sophiesze** [GreenleafLab/10x-scATAC-2019#17]
Hi, 

I really appreciate your work on scATAC-seq.

I tried to run GWAS chromVAR on single-cell summarized experiment using co-accessibility. However,  I was unable to find the dataset of PICS_GWAS_SNPS.gr.rds.

Would it be possible to provide this data? 

Thank you so much!
